### ISSUE TITLE
Computed value in eps_tdoa3d

### ISSUE THREAD

**Issue by sdalu** [ucnl/UCNLNav#13]
In the computed value returned by `eps_tdoa3d` (in rust), as all the intermediate elements are summed squared, shouldn't the square root be returned?

**Comment by AlekUnderwater** [comment]
No, 'cos it's a least **_squares_** method. A residual value is a square root, but the function is to be called from a solver, that minimizes a squares of residuals.
